README.md

```diff
@@ -203,15 +203,21 @@ Your cryptvm now should boot automatically into your fully encrypted system.
 
 ### Troubleshooting
 
-If the keyserver should be unreachable for whatever reason, you will be dropped into the `(initramfs)`-shell after a few minutes. From here you will be able to unlock your encrypted partition manually using a password.
+#### Manual unlocking
+
+If the keyserver should be unreachable for whatever reason, you will be dropped into the `(initramfs)`-shell after a few minutes. From here you will be able to unlock your encrypted partition manually using a password. For this to work it is critical, that *CRYPTSETUP=y* has been set, before creating the initramfs image or else the cryptsetup won't be available in initramfs emergency shell.
```

Comment:
Is that CRYPTSETUP=y part still valid, after removing the other parts about it again?

Reply:
I think so. In theory it should be set automatically, if the system is running a FDE setup, however in Stretch I've found that somewhat unreliable. It seems this parameter is going to be deprecated altogether and replaced by the existence of the *cryptsetup-initramfs* package. I think this could be rephrased to be a little bit more verbose, but I would suggest keeping a reference to the variable in there, so that anybody missing the cryptsetup binaries would know why that probably is.

Reply:
Ok agree.